docs/intro/arch_overview/tracing.rst

```diff
@@ -45,6 +33,46 @@ initiated:
 * Randomly sampled via the :ref:`random_sampling <config_http_conn_man_runtime_random_sampling>`
   runtime setting.
 
+Trace context propagation
+-------------------------
+Envoy provides the capability for reporting tracing information regarding communications between
+services in the mesh. However, to be able to correlate the pieces of tracing information generated by the
+various proxies within a call flow, the services must propagate certain trace context between the inbound
+and outbound requests.
+
+Whichever tracing provider is being used, the service should propagate the
+:ref:`config_http_conn_man_headers_x-request-id` to enable logging across the invoked services
+to be correlated.
+
+The tracing providers also require additional context, to enable the parent/child relationships
+between the spans (logical units of work) to be understood.
+This can be achieved by using the LightStep or Zipkin tracer directly within the service itself,
+to extract the trace context from the inbound request and inject it into any subsequent outbound requests.
```

Comment:
nit: `... outbound requests. (Note: for LightStep, the above is accomplished via OpenTracing; there should be no need to invoke LightStep's tracer directly for extraction or injection of context state) This approach would ...`

(Or any other edit that makes it clear that LightStep's tracer is abstracted underneath OpenTracing's API)

Reply:
I've added `LightStep (via OpenTracing API)` - let me know if that is ok.